Project.toml

```
@@ -34,7 +34,7 @@ GenericSVD = "0.2"
 MacroTools = "0.5"
 MuladdMacro = "0.2.1"
 NLsolve = "4"
-Parameters = "0.11"
+Parameters = ">= 0.10.0"
```

I'm not a fan of `>=` bounds anymore, since any breaking updates might break our packages then. I'd suggest 
```suggestion
Parameters = "0.10, 0.11"
```